forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -197,12 +261,80 @@ pub async fn deploy(command: cmd::Deploy) -> Result<Vec<DeployedContract>> {
                     bail!("Both `--salt` and `--default-salt` were specified: must choose one")
                 }
             };
-            let contract_id =
-                deploy_pkg(&command, &pkg.descriptor.manifest_file, &pkg, salt).await?;
-            contract_ids.push(contract_id);
+            let node_url = get_node_url(&command.node, &pkg.descriptor.manifest_file.network)?;
+            println!(
+                "   {} contract: {}",
```

Comment:
The indentation is off. I think this is right:

```suggestion
                " {} contract: {}",
```

![image](https://github.com/FuelLabs/sway/assets/47993817/71adcd20-6783-4fd8-b377-5ece1929e099)

We should create a helper function for this is forc-tracing. Something that takes the "action" and puts it in bold-green with the right indentation.

Reply:
It was intentionally indented to keep the old style, but happy to change it.